assets/main.scss

```diff
@@ -121,21 +121,18 @@ div.table-42-25-33 table{
   }
 }
 
-div.table-17-23-20-20-20 table{
+div.table-13-29-29-29 table{
```

Comment:
somehow this style name asynced with its content.

Reply:
Yeah I just updated the value insane without changing the class value, will be fixed